.github/workflows/ci.yml

```
@@ -27,9 +27,13 @@ jobs:
       with:
         path: ./src/github.com/linuxkit/linuxkit
 
+    - name: Set path
+      run: echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
```

I guess (looking at `GOPATH: ${{runner.workspace}}` below), this would now be the equivalent of

```suggestion
      run: echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
```

correct? Not sure if there's advantage of one over the other though.

---

Technically, no advantage. Given that we already set `GOPATH: {{runner.workspace}}` in the yaml, I would rather consume what it is set to via `go env GOPATH` than use it explicitly again. Sort of like setting a var and using the var (sort of).

---

gotcha